### PR TITLE
Desktop header style changes

### PR DESCRIPTION
**Pull request by zeftilldeath**
## What does this change?
Mainly visual tweaks based on feedback from Alex, Ben, Stephan, Natalia, Duarte and many more.

Some of the main changes:

- Vertical lines have been removed and horizontal lines added
- Search bar now has a search button on its focus state
- Edition switcher has a label to the left for clarity, and to differentiate it from search field
- Animation of the pillars as the nav is open and closed (Something I want to further improve)
- Hover and focus states 

Before:
screen shot 2017-06-26 at 15 34 22

After:
screen shot 2017-06-26 at 15 34 15

Search active state:
screen shot 2017-06-26 at 15 34 43

Menu hover state:
screen shot 2017-06-26 at 15 34 51

Header open/close animation:
header

Some of these changes may have an effect on browsers that don't support flexbox and as we were working in tandem, @gustavpursche will have to take a look to see if any of my stylistic changes affect anything.  There is more I want to do and will do, but this PR was getting too large and messy. I'll be looking at the slim nav separately, and reviewing proposed changes from another UX review.

### CODE REVIEW

File: common/app/common/NewNavigation.scala [hunk 1]
```diff
@@ -75,7 +75,7 @@ object NewNavigation {
 
     val uk = NavLinkLists(
       List(headlines, ukNews, world, business, environment, tech, politics),
-      List(science, globalDevelopment, cities, obituaries)
+      List(science, cities, obituaries, globalDevelopment)
```
Comment (NataliaLKB) [inline common/app/common/NewNavigation.scala]: You may want to take a look at other editions. AU for example has some long ones.

File: static/src/stylesheets/layout/new-header/_header-cta-item.scss [hunk 2]
```diff
@@ -77,10 +76,10 @@
 
     &:hover,
     &:focus {
-        color: currentColor;
+        color: $guardian-brand-dark;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_header-cta-item.scss]: What do you think about the words going white too?

File: static/src/stylesheets/layout/new-header/_veggie-burger.scss [hunk 3]
```diff
@@ -29,26 +29,34 @@
         right: $gutter-xlarge;
     }
 
+    @include mq(desktop) {
+        &:focus {
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_veggie-burger.scss]: So visually the focus should be on this element, but actually for tab users the focus shouldn't be. See my closed PR for how you should do this: https://github.com/guardian/frontend/pull/17177/files

File: static/src/stylesheets/layout/new-header/_header-cta-item.scss (hunk 2 above)
Comment (zeftilldeath) [inline static/src/stylesheets/layout/new-header/_header-cta-item.scss]: Tried this and it feels a little too extreme. Will review when I look at accessibility changes from UX though 👍

File: static/src/stylesheets/layout/new-header/_menu-item.scss [hunk 4]
```diff
@@ -119,20 +99,26 @@
 
     @include mq(desktop) {
         font-size: 16px;
+        letter-spacing: .02rem;
         line-height: 18px;
-        padding-bottom: $gs-baseline / 2;
+        padding-bottom: 9px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: :laughing: does 8px really not work?

File: static/src/stylesheets/layout/new-header/_menu-item.scss [hunk 5]
```diff
@@ -119,20 +99,26 @@
 
     @include mq(desktop) {
         font-size: 16px;
+        letter-spacing: .02rem;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: can we do px here?

File: static/src/stylesheets/layout/new-header/_menu-item.scss [hunk 6]
```diff
@@ -119,20 +99,26 @@
...
     }
 
     &:hover,
     &:focus {
         text-decoration: none;
 
-        // TODO: maybe unify?
         @include mq(desktop) {
-            color: #ffffff;
+            color: #00d4ff;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: Really? Do we really need a new colour? If so, can we use it as a variable?

File: static/src/stylesheets/layout/new-header/_pillars.scss [hunk 7]
```diff
@@ -35,8 +35,13 @@
     display: inline-block;
 
     @include mq(desktop) {
+        min-width: 0;
+        padding-right: 16px;
+        transition: min-width 600ms ease;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_pillars.scss]: I kind of think this should be quicker, it feels a bit slow to me. What do you think?

File: static/src/stylesheets/layout/new-header/_menu-item.scss (hunk 4 above)
Comment (zeftilldeath) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: Haha. I'm using increments of our baseline. So 3/6/9/12 etc

File: static/src/stylesheets/layout/new-header/_menu-item.scss (hunk 5 above)
Comment (zeftilldeath) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: Sure

File: static/src/stylesheets/layout/new-header/_menu-item.scss (hunk 6 above)
Comment (zeftilldeath) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: We need a colour that is visually brighter/louder than the white, and none of our blues are bright enough 😛

**Comment by gustavpursche** [comment]
@zeftilldeath Since we don't have the vertical borders anymore we can remove almost all `+ 1` and `- 1` calculations, making the CSS a lot more readable.

File: static/src/stylesheets/layout/new-header/_pillars.scss [hunk 8]
```diff
@@ -14,7 +14,7 @@
 
     @include mq(desktop) {
         .new-header--open & {
-            z-index: $zindex-main-menu + 1;
```
Comment (gustavpursche) [inline static/src/stylesheets/layout/new-header/_pillars.scss]: Good catch!

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**